controllers/actions.github.com/resourcebuilder_test.go

```diff
@@ -0,0 +1,92 @@
+package actionsgithubcom
+
+import (
+	"context"
+	"testing"
+
+	"github.com/actions/actions-runner-controller/apis/actions.github.com/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestLabelPropagation(t *testing.T) {
```

Comment:
Not sure how much value is added through this test, but I left it here since I already wrote it :relaxed: 
Let me know if anyone thinks this is useless. It is basically just something to confirm they are passed from one resource to another...